sdk/formrecognizer/ai-form-recognizer/README.md

```diff
@@ -0,0 +1,483 @@
+# Azure Form Recognizer client library for JavaScript
+
+Azure Cognitive Services [Form Recognizer](https://azure.microsoft.com/services/cognitive-services/form-recognizer/) uses cloud-based machine learning to extract structured data from documents. Its features include:
+
+- Custom Document Models: Create machine learning models to extract structured data such as field values and object data structures from documents. The models' document types are trained on your own data, so they're tailored to your documents' structures. Then, use the custom models to extract fields and content from more documents that have the same structure. Use the SDK to manage your models by building, listing, deleting, and copying models;
+- Prebuilt Models: Extract data from certain types of common documents (such as receipts, invoices, business cards, or identity documents) using models developed by the Azure Form Recognizer team.
+- Layout Model: Extract raw document elements&mdash;such as pages (with text words/lines and selection marks), tables, and text styles&mdash;and their bounding regions from input documents.
+- Generic Document Model: Extract entities (categorized text elements) and key-value pairs (associations from one text element, such as a label, to another text element) in addition to the information produced by the prebuilt layout model.
+
+[Source code](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/formrecognizer/ai-form-recognizer/) |
+[Package (NPM)](https://www.npmjs.com/package/@azure/ai-form-recognizer) |
+[API reference documentation](https://docs.microsoft.com/javascript/api/@azure/ai-form-recognizer) |
+[Product documentation](https://docs.microsoft.com/azure/cognitive-services/form-recognizer/) |
+[Samples](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/formrecognizer/ai-form-recognizer/samples)
```

Comment:
I think link needs to be updated to https://github.com/Azure/azure-sdk-for-js/tree/5784af6f121bb59efc2070805e53b4f3c261d2e3/sdk/formrecognizer/ai-form-recognizer/samples-dev ?

Reply:
Once the SDK is released, I'll make publication-ready copies of the samples in samples/v4-beta, so a customer who clicks this link (wherever it might be published out there in the world) will then be given a choice between v3 and v4-beta.